components/footer/footer.jsx

```diff
@@ -1,16 +1,47 @@
 import React from 'react';
-// import { Link } from 'react-router';
+import { MofoFooter } from 'mofo-ui';
 
-export default React.createClass({
-  render() {
-    return (
-      <footer>
-        <div className="container">
-          <p>Get new projects by email. <a id="sign-up-btn" href="https://groups.google.com/a/mozillafoundation.org/d/forum/pulse-notifications/join">Sign up</a>.</p>
-          <p><a href="https://github.com/mozilla/network-pulse">Contribute</a> to this prototype.</p>
-          <p>Submit an <a href="https://docs.google.com/a/mozillafoundation.org/forms/d/e/1FAIpQLScsZ6NLDIo87YH8Z3guR-9slPampPTavcKgbqOJvLsrnqIVMQ/viewform">Impact Story tip</a>.</p>
-        </div>
-      </footer>
-    );
-  }
-});
+const Footer = () => {
+  const FOOTER_LINKS = [
```

Comment:
Um now I think of it, it could be helpful if we have defaults for `cc-license`, `code-of-conduct`, `cookies`, `legal`, and `privacy`. Let's move this discussion to mofo-ui: https://github.com/mozilla/mofo-ui/issues/55